### ISSUE TITLE
UK BA.2 with nuc:T670G = Orf1a:R135S reverted to T/R (potential recombinant?) [7k sequences]

### ISSUE THREAD

**Issue by corneliusroemer** [cov-lineages/pango-designation#546]
In https://github.com/cov-lineages/pango-designation/issues/532 @FedeGueli pointed out that in that lineage nuc:670 =  seemed to have reverted back to wild type (=BA.1).

Trying to see whether #532 was part of a bigger lineage I noticed that BA.2* with nuc:670 reverted are actually on the rise everywhere. They make up 4% of all BA.2 in the UK now, with 7k sequences.

Could it be that this is a recombinant with very early breakpoint, where BA.2 gets rid of a deleterious mutation avoiding Muller's ratchet.

Any BA.1-BA.2 recombinants start with nuc:670 reversion, followed by nuc:2790 reversion. If the recombination breakpoint is between 670 and 2780 all we will see is nuc:670 reverted.

The Usher tree is clean: 
<img width="1204" alt="image" src="https://user-images.githubusercontent.com/25161793/162594221-bf317126-6d4d-462e-b3ef-967387749a94.png">

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_1b4d0_20cea0.json?c=pango_lineage_usher&label=nuc%20mutations:T670G,G4184A,C4321T,C9344T,A9424G,C9534T,C9866T,C10198T,G10447A,C12880T,T15240C,C15714T,C17410T,C19955T,A20055G,C21618T,T21762C,T21846C,T22200G,C22673T,A22688G,G22775A,A22786C,A24130C,C26060T,C26858T,G27382C,A27383T,T27384C,A29510C

I think it would be good to designate this lineage - it has a solid growth advantage over BA.2 in UK

<img width="1665" alt="image" src="https://user-images.githubusercontent.com/25161793/162594234-ef5a7e81-0105-497a-8af8-ffa77f67f3b5.png">

https://cov-spectrum.org/explore/United%20Kingdom/AllSamples/Past6M/variants?nucMutations=670T&pangoLineage=BA.2*&nucMutations1=670T&pangoLineage1=BA.2*&

The lineage proposed in #532 would then be a sublineage - if it was designated at all. I think this issue here should take priority as it is parental.

So this could be recombination  with very early breakpoint? Not sure if there are other mutations that could help us there differentiate between BA.1 and BA.2 - probably not?

We could argue whether we should give this lineage an X letter. While it's plausible that this is a recombinant, it can be explained without recombination. So maybe it's easier to just give it a BA.2 sublineage name.

Actually the same reversion or recombination happens in the parental BA.2 with 9866C. So my bet would be recombination but it's impossible to prove with certainty.

It would be good if someone could have a look and see whether this may be sequencing artefact though.

**Comment by FedeGueli** [comment]
Thx @corneliusroemer Cornelius for looking deeper at it. my bet too is on an early recombination event. have you the tree for parental 9866C with this reversion?

**Comment by FedeGueli** [comment]
@JosetteShoenma  guessed if this tree could be explained with coinfections (she worked on a similar reversion in ndl and the sequencer gave her this explanation)? as we have seem with some different recombinants which Usher tends to put together in the tree could this be an explanation too? 
But in your tree @corneliusroemer  there is a big branch with C2338T and that mutations in just 12 sequences with 670G (orf1a:S135R) and 4500 with 670T (orf1a:R135S) .

So i think.at least  this one with G670T (orf1a:R135S) + C2338T is real.

C2338T was in 50 BA.1 sequence in Uk too not belonging to a single sublineage:
https://cov-spectrum.org/explore/United%20Kingdom/AllSamples/Past6M/variants?nucMutations=C2338T%2CG670T&pangoLineage=BA.1*&

**Comment by corneliusroemer** [comment]
Yes @FedeGueli I agree, I've also spotted that big C2338T branch.

![image](https://user-images.githubusercontent.com/25161793/163264645-426f80ab-5c8b-457b-9b78-c6c8b1e072d3.png)

I agree it's worth designating, it's very clearly most common in the UK, appearing in other countries, but only much later.

Interestingly, there's a BA.1* lineage with C2338T present that may have been BA.1 donor. It would explain the two mutations with one recombination event - rather than 2 single point mutation. Hard to say anything for sure - so I'd just give it a BA.2.x label.

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_29af_72a7a0.json?label=nuc%20mutations:T670G,C2790T,G4184A,C4321T,C9344T,A9424G,C9534T,C9866T,C10198T,G10447A,C12880T,T15240C,C15714T,C17410T,C19955T,A20055G,C21618T,T21762C,T21846C,T22200G,C22673T,A22688G,G22775A,A22786C,A24130C,C26060T,C26858T,G27382C,A27383T,T27384C,A29510C

**Comment by chrisruis** [comment]
Thanks @corneliusroemer Tricky to assess whether this is a reversion or recombination event. I think it's best to add this as a BA.2 lineage. We've added this as BA.2.23 to start on the branch with G670T (Orf1ab:R135S) which corresponds with the introduction(s) into the UK
